src/patternfly/demos/Card/examples/CardView.css

```diff
@@ -0,0 +1,25 @@
+#ws-html-demos-c-card-status-card-expanded-with-popover {
+  padding-top: 180px;
+}
+
+#ws-html-demos-c-card-status-card-expanded-with-popover #carview-demo--popover-parent,
+.popover-parent {
+  position: relative;
+}
+
+#ws-html-demos-c-card-status-card-expanded-with-popover .pf-c-popover {
+  position: absolute;
+  transform: translate(130px, -50%);
```

Comment:
I think the magic sauce for this to be relatively positioned (instead of fixed 130px), is:

```
transform: translateY(-50%);
left: calc(100% + var(--pf-c-popover__arrow--Width) + var(--pf-global--spacer--sm));
```